packages/material-ui-icons/scripts/download.js

```diff
@@ -1,42 +1,47 @@
 /* eslint-disable no-console */
 
+import 'isomorphic-fetch';
 import fse from 'fs-extra';
 import path from 'path';
 import yargs from 'yargs';
 import Queue from 'modules/waterfall/Queue';
 import sleep from 'modules/waterfall/sleep';
 import retry from 'modules/waterfall/retry';
-import 'isomorphic-fetch';
 
 const themeMap = {
   baseline: '', // filled
-  outline: '_outlined',
-  round: '_rounded',
-  twotone: '_two_tone',
-  sharp: '_sharp',
+  outline: 'outlined',
+  round: 'round',
+  twotone: 'twotone',
+  sharp: 'sharp',
+};
+
+const themeFileNameMap = {
+  baseline: '', // filled
+  outline: 'Outlined',
+  round: 'Rounded',
+  twotone: 'TwoTone',
+  sharp: 'Sharp',
 };
 
 function downloadIcon(icon) {
-  console.log(`downloadIcon ${icon.index}: ${icon.id}`);
+  console.log(`downloadIcon ${icon.index}: ${icon.name}`);
 
   return Promise.all(
     Object.keys(themeMap).map(async theme => {
-      let endUrl;
-      if (icon.imageUrls && icon.imageUrls[theme]) {
-        endUrl = icon.imageUrls[theme];
-      } else {
-        endUrl = `${theme}-${icon.id}-24px.svg`;
-      }
-      const size = endUrl.match(/^.*-([0-9]+)px.svg$/)[1];
-      const response = await fetch(`https://material.io/tools/icons/static/icons/${endUrl}`);
+      const response = await fetch(
+        `https://fonts.gstatic.com/s/i/materialicons${themeMap[theme]}/${icon.name}/v${
+          icon.version
+        }/24px.svg`,
+      );
       if (response.status !== 200) {
         throw new Error(`status ${response.status}`);
       }
       const SVG = await response.text();
       await fse.writeFile(
         path.join(
           __dirname,
-          `../material-io-tools-icons/ic_${icon.id}${themeMap[theme]}_${size}px.svg`,
+          `../material-io-tools-icons/ic_${icon.name}${themeFileNameMap[theme]}_24px.svg`,
```

Comment:
We could drop the` ic_` prefix and update the rename filter.

While I'm being pedantic, `material-io-tools-icons` doesn't match the current (website) URL: https://material.io/resources/icons

Reply:
How about we just make it generic: `material-icons`

Reply:
`material-io-tools-icons` is now `material-icons`